app/models/manageiq/providers/ibm_cloud/inventory/collector/power_virtual_servers.rb

```diff
@@ -38,4 +38,16 @@ def ports(network_id)
   def sshkeys
     connection.get_ssh_keys
   end
+
+  def system_pool
+    connection.get_system_pool.values
+  end
+
+  def storage_types
+    connection.get_storage_types[region]
+  end
+
+  def region
+    manager.provider_region || "us-south"
```

Comment:
We can look at this in a follow-up but I think `provider_region` is required (could be added as a validate presence to the cloud_manager) so the `|| "us-south"` shouldn't be needed.  Would rather enforce that it be set than handle the possibility that it might be `nil` everywhere.

Reply:
#40 removes this region method

Reply:
sure will do it.

Reply:
We can still set the `provider_region` at registration time, especially if it's used for UI elements.